.gitsubprojects

```diff
@@ -2,3 +2,5 @@
 git_subproject(Servus https://github.com/HBPVIS/Servus.git 567f038)
 git_subproject(Lunchbox https://github.com/Eyescale/Lunchbox.git b41edcf)
 git_subproject(vmmlib https://github.com/Eyescale/vmmlib.git b41e1a5)
+git_subproject(mvd-tool ssh://bbpcode.epfl.ch/hpc/MVD-tool 517ca7f)
```

Comment:
Such a huge change, I'm not sure it will be possible :D 
no problem
Can you associate it with the HPC account of github ?